client/me/help/help-happiness-engineers/index.jsx

```diff
@@ -1,57 +1,61 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	PureRenderMixin = require( 'react-pure-render/mixin' );
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import React from 'react';
+import PureRenderMixin from 'react-pure-render/mixin';
 
 /**
  * Internal dependencies
  */
-var HappinessEngineersStore = require( 'lib/happiness-engineers/store' ),
-	HappinessEngineersActions = require( 'lib/happiness-engineers/actions' ),
-	FormSectionHeading = require( 'components/forms/form-section-heading' ),
-	Gravatar = require( 'components/gravatar' );
+import FormSectionHeading from 'components/forms/form-section-heading';
+import Gravatar from 'components/gravatar';
+import QueryHappinessEngineers from 'components/data/query-happiness-engineers';
+import { getHappinessEngineers } from 'state/happiness-engineers/selectors';
 
-module.exports = React.createClass( {
-	displayName: 'HelpHappinessEngineers',
-
-	mixins: [ PureRenderMixin ],
-
-	componentDidMount: function() {
-		HappinessEngineersStore.on( 'change', this.refreshHappinessEngineers );
-		HappinessEngineersActions.fetch();
-	},
-
-	componentWillUnmount: function() {
-		HappinessEngineersStore.removeListener( 'change', this.refreshHappinessEngineers );
-	},
-
-	getInitialState: function() {
-		return {
-			happinessEngineers: HappinessEngineersStore.getHappinessEngineers()
-		};
-	},
-
-	refreshHappinessEngineers: function() {
-		this.setState( { happinessEngineers: HappinessEngineersStore.getHappinessEngineers() } );
-	},
-
-	render: function() {
-		if ( this.state.happinessEngineers.length === 0 ) {
-			return null;
-		}
+class HelpHappinessEngineers extends React.Component {
```

Comment:
This component is a good candidate to be written as a [stateless function component](https://facebook.github.io/react/docs/components-and-props.html) since it doesn't rely on component lifecycle.